### ISSUE TITLE
siteurl instead of domain

### ISSUE THREAD

**Issue by GutHib** [WebDevStudios/WDS-Active-Plugin-Data#10]
Hi,

Nice little plugin you wrote there. Just a small suggestion - in line 253, you are using $site->domain to come up with an identifier for each site, but at least for me, that just gets you the main domain every time. $site->siteurl does the trick.

Thanks!

**Comment by tw2113** [comment]
Interesting. At the same time though, the only part we use that for is a title attribute, which arguably could also be a candidate for removal as a whole.

**Comment by GutHib** [comment]
Yeah, but it's actually quite useful, because if you couldn't hover over the numbers to have your URL displayed, how could you easily figure out which column relates to which site?

**Comment by tw2113** [comment]
Output that info as part of the link text. Been too long since I looked at the actual output to see if that'd break a lot of visual layout and whatnot. Just throwing the thought out there.

**Comment by GutHib** [comment]
I think you decided against that to keep the columns slim. If you put more than a number, I would assume it indeed breaks the layout.

**Comment by tw2113** [comment]
Probably. Regardless, make whatever changes you need to make it work for you. PRs welcome as well.

**Comment by GutHib** [comment]
Already did that. Just mentioned it in case somebody wants to incorporate it as a fix into the "official" script. Not worth the hassle to make my own commit, since I'm not a dev nobody is gonna use that, ever.

**Comment by tw2113** [comment]
Handled in 3a123dfb38a31298d2361c561f078266472c9aa9
